modules/ROOT/pages/_partials/data-sync/js-client-mutate.adoc

```diff
@@ -60,3 +61,38 @@ It should include an array similar to the following:
   }
 }
 ----
++
+. Browse http://localhost:4000/graphql and enter the playground query for the addressbook. For example:
++
+[source,json]
+----
+{
+  addressBook {
+    name
+    address
+
+}
+----
++
+Results should be similar to:
++
+[source,json]
+----
+{
+  "data": {
+    "addressBook": [
+      {
+        "name": "Alex Smith",
+        "address": "1 Square Place, City"
+      },
+      {
+        "name": "John Doe",
+        "address": "1 Red Hill"
+      }
+    ]
+  }
+}
+
+  }
```

Comment:
Need to remove line 96 and 97.  There are 2 extra curly braces that render the json invalid.